Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1490,6 +1488,17 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
     }
   };
 
+  function multiplyNonNullMatrices(keyMat, actMat, inverseShiftScaleMat) {
```

Comment:
This is a counter intuitive function name because you actually actMat and inverseShiftScaleMat can be null.

Rename to `safeMat4Multiply()` ?

Reply:
Maybe this function could be more generic e.g. safeMat4Multiply(m1, m2, m3) or even better, take an undefined number of matrices as input and multiply them all